Changefeed/ExampleEventCreator/ExampleEventCreator.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Azure.Storage.Blobs;
+using System;
+using System.IO;
+using System.Text;
+using System.Configuration;
+using System.Threading;
+
+namespace ExampleEventCreator
+{
+    class ExampleEventCreator
+    {   
+        static int count;
+        // Storage account connection string
+        static string connectionString = ConfigurationManager.AppSettings["connectionString"];
+        // Name of created container
+        static string containerName = "test-changefeed-container";
+
+        // Creates container and blobs within a storage account to populate changefeed and add example events
+        static void Main(string[] args)
+        {
+            count = 0;
+            // Create clients
+            BlobServiceClient blobServiceClient = new BlobServiceClient(connectionString);
+            BlobContainerClient blobContainerClient = blobServiceClient.CreateBlobContainer(containerName);
+
+            // Create Timer
+            Timer timer = null;
+            Console.WriteLine("Press Space to start the timer. Press Enter to stop the timer.");
+            if(Console.ReadKey().Key == ConsoleKey.Spacebar)
+            {
+                Console.WriteLine("Starting timer...");
+                // Timer calls CreateEvents function every 30 minutes
+                timer = new Timer(new TimerCallback(CreateEvents), null, 0, 1800000);
+
+                while (Console.ReadKey().Key != ConsoleKey.Enter)
+                {
+                    if (Console.ReadKey().Key == ConsoleKey.Enter)
+                    {
+                        Console.WriteLine("Stopping timer...");
+                        timer.Change(
+                            Timeout.Infinite,
+                            Timeout.Infinite);
+                        break;
+                    }
+                }
+
+                Console.WriteLine("Timer Stopped");
+            }                             
```

Comment:
I probably should have suggested this sooner.

We should have an else to tell the user we didn't start the timer and exitted the program since they didnt press space. Also update the  "Press Space to start the timer. Press Enter to stop the timer." to add on "To exit / not run the timer, press any other key". 
---
Or maybe my other suggestion is for the user to enter the amount of time for the timer to run instead of the user deciding to press a key to stop it.